lib/inspec/resources/postgres_session.rb

```diff
@@ -40,7 +40,7 @@ class PostgresSession < Inspec.resource(1)
       end
     EXAMPLE
 
-    def initialize(user, pass, host = nil, port = nil)
+    def initialize(user, pass = nil, host = nil, port = nil)
       @user = user || "postgres"
       @pass = pass
       @host = host || "localhost"
```

Comment:
port seems to be missing here?

Reply:
port is there if port is not defined by user it will use default 5432. I just need to remove that pass = nil as those changes are not required and the password should be mandatory.